app/controllers/school_search_controller.rb

```diff
@@ -1,4 +1,7 @@
-class SchoolSearchController < BasePublicController
+class SchoolSearchController < ApplicationController
+  include DfE::Analytics::Requests
+  include HttpAuthConcern
```

Comment:
Can I just check why we are not using the `BasePublicController` here?

Reply:
School search sits outside of the journey routes so `journey` will resolve to `nil`. `BasePublicController` assumes it is used in the journey context and does some things like altering the view path which won't work and is not needed here.